assets/helpers/user/user.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { setDisplayName, setEmail, setFirstName, setLastName, setTestUser, setFullName } from './userActions';
+import * as cookie from './../cookie';
+
+export default function (dispatch: Function) {
```

Comment:
I would name this function as init and then include it as an object and use it as user.init(). The named functions give us more readability and in debuggers appears the name so it is easier to track an error.